stubs/openpyxl/openpyxl/xml/_functions.pyi

```diff
@@ -0,0 +1,59 @@
+# This file does not exist at runtime. It is a helper file to overload imports in openpyxl.xml.functions
+# TODO: Add missing types in lxml-stubs and et_xmlfile and figure out how to import lxml-stubs
```

Comment:
Got my answer, definitely not in favor of type hints or the type system at all:
https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1553#note_145934
https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1570#note_150282
https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1812#note_183907
https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1902#note_190424
https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1922#note_192543

So that'd be a hard sell. If they don't want to even link to `types-openpyxl`, I doubt they'd add an extra dep to `lxml` if the only "users" asking for it are typeshed contributors.

At least I'll stop entertaining that idea 😅